debian/copyright

```diff
@@ -1,16 +1,24 @@
-This package is currently maintained by Christoph Hueffelmann <chr@istoph.de> 2016 
+This package is actually maintained by Christoph Hüffelmann <chr@istoph.de> 2016
```

Comment:
Change the copyright format to match the format from Debian's packaging manual: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/